script.js

```diff
@@ -97,11 +97,7 @@ const myProjects = [];
 
 myProjects.push(
   new ProjectCard(
-<<<<<<< HEAD
     './img/img1.png',
-=======
-    'img/img1.png',
->>>>>>> f69eacf89ac65d382729aa20b2fbd3694dbe0859
```

Comment:
- [ ] This script.js file is full of merge conflicts. please fix it.

Reply:
![Sin título](https://user-images.githubusercontent.com/54015740/124048298-9ec97480-d9db-11eb-95f0-a601870e7f0a.png)
That's not true, I checked thoughtfully before sending my pull request and I don't have any merging errors, and it's all working on the clean tree, plus I fixed those merging errors, it was from the previous branch.  I am providing screenshot proof also all checks have passed before I even sent my pull request 
![image](https://user-images.githubusercontent.com/54015740/124048387-dcc69880-d9db-11eb-8aaa-1e48c897049b.png)

Reply:
![Sin título](https://user-images.githubusercontent.com/54015740/124048720-96be0480-d9dc-11eb-8ae2-2c28708c41e6.png)

I checked it many times, and I think you made a mistake, we are talking about the current branch validate-contact form, this branch doesn't have merging mistakes, the one that got some is main, and it will be corrected when I merge this pull request after approval. 
![image](https://user-images.githubusercontent.com/54015740/124048822-ca009380-d9dc-11eb-9ca7-447c45443fe6.png)

With these screenshots I prove that this branch is alright, and doesn't present any errors of no type, no merging errors either, what you saw on this code quote is from a previous branch.

Reply:
It even says working on clean tree